examples/k8s-upgrade/v1.7.9-hybrid.json

```diff
@@ -3,7 +3,7 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorVersion": "1.7.9"
```

Comment:
Do we drop windows support for 1.7? If so, please rename the file to examples/k8s-upgrade/v1.8.8-hybrid.json and update test config test/acse-conf/acse-upgrade-win.json

Reply:
I didn't realize this was testing upgrade, I reverted the change. Thanks!